python/dicebeard/__init__.py

```diff
@@ -53,17 +58,49 @@ def __init__(self, *args, **kwargs):
         self.my_dice = dice.Dice(self.images_path, self.font_path)
         self.my_coin = coin.Coin(self.images_path, self.font_path)
 
+
+    _timeout = 90
+
+    @onerror()
+    async def train(self,msg):
+        '''Game for training adding up dice.'''
+        #Outputs a BytesIO stream and the total value of the dice
+        input_args = get_args(msg)
+        i = 0
+        
+        try:
+            i = int(input_args[0])
+            i = 10 if i > 10 else i
+        except Exception:
```

Comment:
Using generic `Exception` can lead to problems, it's better to figure out what the specific exception is and then use that. (If you don't know what exception it could be, then you're really just covering up what could be a nasty bug)